src/elements/content-uploader/ItemName.scss

```diff
@@ -1,7 +1,7 @@
 @import '../common/variables';
 
 .bcu-item-label {
-    max-width: 300px;
+    width: 100%;
```

Comment:
removing `max-width` will support longer file names in large screen sizes

**Before**
<img width="970" alt="Screenshot 2023-12-29 at 12 23 27 PM" src="https://github.com/box/box-ui-elements/assets/7311041/e53e778c-f90d-4320-8596-4e4c9ae850d9">

**After**
<img width="968" alt="Screenshot 2023-12-29 at 12 24 16 PM" src="https://github.com/box/box-ui-elements/assets/7311041/f5a91320-6e77-4933-af57-bf6ac4d1fd72">
<img width="970" alt="Screenshot 2023-12-29 at 12 24 57 PM" src="https://github.com/box/box-ui-elements/assets/7311041/02568380-d4b1-4ae4-9214-199e6fe9bc30">